Druid-Tests/DRIRTest.class.st

```diff
@@ -41,10 +41,27 @@ DRIRTest >> assertConstraint: aConstraint includes: includedConstraint [
 	self assert: (aConstraint includes: includedConstraint)
 ]
 
+{ #category : #asserting }
+DRIRTest >> assertConstraintIn: aDRCPSEdge for: aKey includes: expectedConstraint [
+
+	| constaint |
+	constaint := aDRCPSEdge constraintFor: aKey.
```

Comment:
what is a key? a (register) name?

Reply:
Yes, the string. For now, it was enough.

Reply:
ok!